src/coreclr/tests/helixpublishwitharcade.proj

```diff
@@ -5,7 +5,7 @@
        to send test jobs to helix. -->
 
   <Import Sdk="Microsoft.DotNet.Helix.Sdk" Project="Sdk.props" Condition=" '$(UsesHelixSdk)' == 'true' " />
-  <Import Project="Directory.Build.props" Condition=" '$(UsesHelixSdk)' != 'true' "/>
+  <Import Sdk="Microsoft.Build.NoTargets" Project="Sdk.props" Condition=" '$(UsesHelixSdk)' != 'true' " />
```

Comment:
I don't think the `UsesHelixSdk` property is ever defined. You can remove the line above, remove this condition, move the sdk import into the project tag and remove the targets import at the bottom.

Reply:
OK, ignore my comment, it is passed in in L59 below in the "Properties" tag